src/Components/Facility/HospitalList.tsx

```diff
@@ -83,10 +81,11 @@ export const HospitalList = () => {
                 <div className="flex py-4 justify-between">
                   <div>
                     <div className="text-gray-500 leading-relaxed">Phone:</div>
-                    <div className="font-semibold">{facility.phone_number || "-"}</div>
+                    <a href={`tel:${facility.phone_number}`} className="font-semibold">{facility.phone_number || "-"}</a>
                   </div>
                   <span className="inline-flex rounded-md shadow-sm">
-                    <button type="button" className="inline-flex items-center px-3 py-2 border border-green-500 text-sm leading-4 font-medium rounded-md text-green-700 bg-white hover:text-green-500 focus:outline-none focus:border-green-300 focus:shadow-outline-blue active:text-green-800 active:bg-gray-50 transition ease-in-out duration-150">
+                    <button type="button" className="inline-flex items-center px-3 py-2 border border-green-500 text-sm leading-4 font-medium rounded-md text-green-700 bg-white hover:text-green-500 focus:outline-none focus:border-green-300 focus:shadow-outline-blue active:text-green-800 active:bg-gray-50 transition ease-in-out duration-150 hover:shadow"
+                        onClick={() => navigate(`/facility/${facility.id}`)}>
```

Comment:
moved onClick to here. To be consistent with most places where only on clicking button inside card takes you to the page inside. And for phone links to be clickable.